ee/desktop/runtime/runner_test.go

```diff
@@ -20,13 +18,22 @@ import (
 func TestDesktopUserProcessRunner_Execute(t *testing.T) {
 	t.Parallel()
 
-	// When running this using the golang test harness, it will leave behind proccess if you do not build the binary first.
+	if runtime.GOOS == "linux" {
+		t.Skip("skipping linux test because it's not implemented")
+	}
+
+	// When running this using the golang test harness, it will leave behind process if you do not build the binary first.
 	// On mac os you can find these by setting the executable path to an empty string before running the tests, then search
 	// the processes in a terminal using: ps aux -o ppid | runtime.test after the tests have completed, you'll also see the
 	// CPU consumtion go way up.
 
 	// To get around the issue mentioned above, build the binary first and set it's path as the executable path on the runner.
-	executablePath := filepath.Join(t.TempDir(), "desktop")
+	executablePath := filepath.Join(t.TempDir(), "desktop-test")
+
+	if runtime.GOOS == "windows" {
+		executablePath = fmt.Sprintf("%s.exe", executablePath)
+	}
+
 	err := exec.Command("go", "build", "-o", executablePath, "../../../cmd/launcher").Run()
 	require.NoError(t, err)
```

Comment:
FWIW to get a useful error, I started doing things like:
```go
                                ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
                                defer cancel()
                                cmd := exec.CommandContext(ctx, boxerRB, "encode", rubyInFile, rubyOutFile)
                                out, err := cmd.CombinedOutput()
                                require.NoError(t, err, string(out))
```

Reply:
cool, added this to tests as well as some fancy log comparison